src/server.js

```diff
@@ -9,6 +9,16 @@ const fs = require('fs');
 // load additional environment variables (ex: database and storage bucket passwords) from .env file
 env.config();
 
+// MongoDB database connection code
+const { MongoClient, ServerApiVersion } = require('mongodb');
+const uri = "mongodb+srv://admin:webarcade7@cluster0.rporgjs.mongodb.net/?retryWrites=true&w=majority";
```

Comment:
It is not secure to store passwords in code. Store this in an environment variable called "MONGO_DB_URI", then retrieve it using process.env.MONGO_DB_URI

Add the following to your .env file: https://docs.google.com/document/d/1F6kL7zoONb92WJbcmlhfFXMjz2BXWTEyaqaB1gwTrb8/edit